lib/content._coffee

```diff
@@ -47,11 +44,14 @@ getFilePathForId = (id) ->
     id = id.replace /([A-Z])/g, '_$1'
     Path.join DIR_BY_ID_PATH, "#{id}.json"
 
-getFilePathForURL = (url) ->
-    if idFile = DATABASE["#{hashURL url}.json"]
-        Path.join DIR_BY_ID_PATH, idFile.replace '../content-by-id', ''
-    else
-        ''
+readFileForURL = (url, _) ->
+    cloudClient.download
+        container: 'content/content-by-url'
+        remote: "#{hashURL url}.txt"
+
+    # Somehow after the download we get the ID and then ...
```

Comment:
This is where my understanding of Streamline + PkgCloud runs out. I guess cloudClient.download returns a pipe or a stream or something? I just copied the code from: 

https://github.com/pkgcloud/pkgcloud#download-a-file

...but I don't know anything about pipes or streams. At any rate, we need to get the data from the pipe/stream and use it (as `id`) in the line below.